src/domain/shared/layout/CardsLayout/CardsLayout.tsx

```diff
@@ -43,7 +43,9 @@ const CardsLayout = <Item extends Identifiable | null | undefined>({
 
   return (
     <CardLayoutContainer {...layoutProps}>
-      {showCreateButton && <CreateButton onCreateButtonClick={onCreateButtonClick}>Add</CreateButton>}
+      {showCreateButton && (
+        <CreateButton onCreateButtonClick={onCreateButtonClick} createButtonWidth={createButtonWidth} />
```

Comment:
I think you should provide the button as a component instead.
Composition over inheritance.

Reply:
Merging to dev, to be re-verified after all-hands session